bin/build-hdfs

```diff
@@ -91,3 +92,13 @@ cd ..
 echo "Compressing to $DIST.tgz"
 rm -f $DIST.tgz
 tar czf $DIST.tgz $DIST
+
+
+# Build docker container
+if [ "$1" == "docker" ] && [ -f $DIST.tgz ]; then
+    if [ ! -z "$2" ]; then
+        DOCKER_IMAGE_NAME=$2
+    fi
+    docker build -t $DOCKER_IMAGE_NAME .
```

Comment:
If you pass in a tag it will run it now like when you run docker build:

`docker build -t repo/image:tag .`